e2e/jest.config.js

```diff
@@ -0,0 +1,11 @@
+/** @type {import('jest').Config} */
+module.exports = {
+  setupFilesAfterEnv: ['./setup.js'],
+  reporters: ['detox/runners/jest/reporter'],
+  globalSetup: 'detox/runners/jest/globalSetup',
+  globalTeardown: 'detox/runners/jest/globalTeardown',
+  testEnvironment: 'detox/runners/jest/testEnvironment',
+  testTimeout: 120000,
+  testMatch: ['./**/*.e2e.js'],
```

Comment:
Can we use the default here? 
```suggestion
  rootDir: '..',
  testMatch: **['<rootDir>/e2e/**/*.e2e.js'],**
```